hic2cool/hic2cool_utils.py

```diff
@@ -736,41 +743,128 @@ def write_zooms_for_higlass(h5res):
             higlass_compat = False
             break
     if not higlass_compat:
-        print('WARNING: This hic file is not higlass compatible! Will not add [max-zoom] attribute.')
+        print('!!! WARNING: This hic file is not higlass compatible! Will not add [max-zoom] attribute.')
```

Comment:
You could print logging messages to `sys.stderr`.

Reply:
Not a huge deal though, since there is currently no meaningful stdout output.

Reply:
Good call. Error messages are printed to sys.stderr, so I might as well do the same for warning messages

Reply:
You might want to define a logging function to reduce the noise of appending `file=sys.stderr` everywhere. There is, of course, the `logging` module too, but it's kind of overkill.

Reply:
Agreed, thanks for the feedback. Defined a logging method and also added some tests